app/views/results/index.html.erb

```diff
@@ -43,6 +43,9 @@
             <% end %>
           <% end %>
 
+          <p class="govuk-body">
+            <%= govuk_link_to 'New search', root_path, no_visited_state: true %>
+          </p>
```

Comment:
@paulrobertlloyd I think this was removed in `master`?

Reply:
at  4356b82a8f65d321485aec4cc3d9cd5f1ddba

Reply:
It was. Thought I removed it in my latest rebase. Did I not?